serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -32,8 +32,11 @@ For more information about the tooling and the required dependencies, see xref:g
 To create your Serverless Workflow service, first you need to bootstrap a project.
```

Comment:
In the `Prerequisites`, please mention that either Quarkus CLI or KN CLI can be used.